src/cljs/doit/events.cljs

```diff
@@ -15,7 +15,10 @@
 
 (defn get-todo-success
   [db [_ todos]]
-  (assoc db :todos todos))
+  (let [todo-id-map (reduce (fn [m t] (assoc m (:id t) t))
+                            {}
+                            todos)]
+    (assoc db :todos todo-id-map)))
```

Comment:
`reduce` to me normally implies there's some "reduction" going on. Since this operation is effectively keeping the map the same, and changing the structure, I'd prefer to do this:

```clojure
(->> (map (fn [t] [(:id t) t]) todos)
     (into {}))
```